app/src/main/java/com/x8bit/bitwarden/data/vault/datasource/sdk/VaultSdkSourceImpl.kt

```diff
@@ -129,7 +129,7 @@ class VaultSdkSourceImpl(
                 InitializeCryptoResult.Success
             } catch (exception: BitwardenException) {
                 // The only truly expected error from the SDK is an incorrect key/password.
-                InitializeCryptoResult.AuthenticationError
+                InitializeCryptoResult.AuthenticationError(message = exception.localizedMessage)
```

Comment:
Have you been able to validate this with a test account?

I am concerned that `localizedMessage` could be null and we just need the `message`

Reply:
This is a valid concern. I went back and forth on it since there is so much localization in the app. Along with this documentation 
> Creates a localized description of this throwable. Subclasses may override this method in order to produce a locale-specific message. For subclasses that do not override this method, the default implementation returns the same result as getMessage().
Returns:
The localized description of this throwable.
Since:
1.1

I felt like this may be the call, however since I really have no way of recreating this other than forcing the error in code by adding this on line 130 in `VaultSdkSourceImpl`
```
throw BitwardenException.E(message = "Nyet")
```
and I see that message returned in the error dialog.

I have no issues changing to just `message` though.

Reply:
The SDK doesn't handle any localization, so `localizedMessage` is always going to return the same as `message`. I'd go with `message` just to be safe. If the message needs to be localized that's something we would have to implement in the app.